tests/freezer-spec.js

```diff
@@ -123,6 +123,38 @@ describe("Freezer test", function(){
 		assert.notEqual( updated, data );
 	});
 
+	it( "Set a value to null", function(){
+		data.set('a', null);
+
+		var updated = freezer.getData();
+
+		assert.deepEqual( updated.a, null );
+
+		assert.notEqual( updated, data );
+	});
+
+	it( "Update a numerical value to null", function(){
+		data.set('a', 1);
+		data.set('a', null);
```

Comment:
@arqex you're right below, but NOT in this:
> The numeric example is passing just because you are setting the value `1` to an attribute that already has a value of `1`, so no update is triggered.

So, what's the difference?